### PR TITLE
Tickets/preops 528

### PR DESCRIPTION
**Pull request by leannep**
I have added a guide to executing ADQL queires in the Portal and given several example queries. I did not repeat the Results Analysis descriptions given in the 'Single table Query' as once the query is exeecuted from the ADQL page, the results view is the same - so I referred to the 'Single table Query' section.

I reorganzied some of the information in the 'Single table Query' section - please review and accept/reject.

Note that as table JOIN is not working in the Portal at the moment, I cannot test the join query. I think it's correct but it should be tested when the problem is fixed.

### CODE REVIEW

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst:61]: In the above section, two small suggestions:
"right ascencion" --> "right ascension", and "The examples under the box for coordinates are object names are examples of the formatting only." --> "The examples under the box for coordinates are object names as examples of the formatting only."

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "simulated DC2 images" --> "simulated DC2 image"

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: This sentence got jumbled? (And the ref doesn't display correctly.)

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: Is **3.** meant to be a reference to a figure/section? It's appearing simply as bold text.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "in the sam manner" --> "in the same manner"

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "tables" --> "table"

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "desireable" --> "desirable"?

Also, should references to columns such as objectIds be placed within backticks?

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: This line cuts off when displayed on my browser. May have to break it into two lines.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "...so that we In the truth_match"? I think the first sentence got cut off.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "availabe" --> "available"

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: "the the"

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst:189]: Would it make sense to include screen shots showing the results of these schema queries? The additional context might be helpful.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: This sentence got jumbled, and the reference isn't displaying correctly.

**Review by jeffcarlin — COMMENTED** [review]
Overall this looks great! Most of the comments below are typos, etc.

We should make sure the table join works correctly before merging. Would it also make sense to make an xy plot in the results page comparing the fluxes between the `object` and `truth_match` tables? (i.e., a similar figure to what is in tutorial notebook 06?)

**Comment by MelissaGraham** [comment]
Note there's a lingering TODO to either be deleted or done. _"(TODO: How do ‘is_good_match and match_object_id work together?)"_

I can't tell what's been reorganized under 'Single Table Query' so it must be fine...

This is covering all the needed bases I think so merge when you're ready!

**Review by MelissaGraham — APPROVED** [review]

**Comment by leannep** [comment]
> Note there's a lingering TODO to either be deleted or done. _"(TODO: How do ‘is_good_match and match_object_id work together?)"_
> 
> I can't tell what's been reorganized under 'Single Table Query' so it must be fine...
> 
> This is covering all the needed bases I think so merge when you're ready!

yes - Jeff and I have worked through trying to understand this. The TODO will be removed

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst]: Yes - it is the section that appears on the Portal when 'ADQL; is selected in 2.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst:189]: yes !

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: The phrasing of this sentence sounds strange. Also, the reference isn't rendering correctly at https://dp0-1.lsst.io/v/PREOPS-528/data-access-analysis-tools/index-portal-intro.html

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst:58]: The link to the DC2 paper isn't rendering correctly at https://dp0-1.lsst.io/v/PREOPS-528/data-access-analysis-tools/index-portal-intro.html

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: What is "...the interface in **3.**..." referring to? It just shows as a bold-faced "3." in the text. Is it a section? Figure?

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [reply data-access-analysis-tools/index-portal-intro.rst]: Don't forget to update "tables" --> "table"

Also, does this need a new section number? 4? or 3.1?

File: data-access-analysis-tools/index-portal-intro.rst
Comment (jeffcarlin) [inline data-access-analysis-tools/index-portal-intro.rst]: Add period at end of sentence.

**Review by jeffcarlin — APPROVED** [review]
I didn't comment on the rsp-warnings section, because that seems like ongoing discussion.

I "approve", but there are a couple small edits suggested.

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst]: I don;t think so because these queries also get input into the 'ADQL queries" interface

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst]: yes

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst]: Section 3. of the Portal Interface

File: data-access-analysis-tools/index-portal-intro.rst
Comment (leannep) [reply data-access-analysis-tools/index-portal-intro.rst:58]: I'm going to leave this to Melissa or Matt to look at. I inherited this from the main branch.

**Pull request merged**
